Cargo.toml

```diff
@@ -45,6 +45,13 @@ members = [
     "testing/separated/*/",
 ]
 
+# Don't include git submodules in our workspace, since those packages haven't
+# been written with that in mind. They might work, but they might not.
+exclude = [
+    "components/external/glean/",
+    "components/external/nimbus-sdk/",
+]
```

Comment:
By excluding these from the Cargo workspace, I appear to be able to use a path dependency for the Rust code rather than fetching it from crates.io, which in theory means there's less chance of us using incompatible versions of the Native code and Kotlin code.